opencga-app/app/scripts/azure/arm/mongodb/postdeploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#this script is wrapped into the mongodb template in base64
+#just here for reference
+
+echo $APP_DNS_NAME
+echo $MONGODB_USERNAME
+echo $MONGODB_PASSWORD
+sudo apt-get update
+sudo apt-get upgrade -y
+sudo apt-get install software-properties-common -y
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
+echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.6.list
+sudo apt-get update
+sudo apt-get install -y mongodb-org
+sudo systemctl enable mongod.service
+sudo add-apt-repository -y universe
+sudo add-apt-repository -y ppa:certbot/certbot
+sudo apt-get update
+sudo apt-get install -y nginx certbot python-certbot-nginx
+sleep 30
+sudo certbot --nginx -d ${APP_DNS_NAME} -m opencga@test.com --agree-tos -q
```

Comment:
We should probably take in a valid email on this one as will likely validate TOS for lets encrypt.

Reply:
Added in https://github.com/opencb/opencga/pull/975/commits/edaf7081723cd925a055f3d855af57370e0debe5